frontend/src/views/app-store/installed/detail/index.vue

```diff
@@ -259,7 +259,7 @@ const get = async () => {
         }
         paramModel.value.memoryLimit = res.data.memoryLimit;
         paramModel.value.cpuQuota = res.data.cpuQuota;
-        paramModel.value.memoryUnit = res.data.memoryUnit !== '' ? res.data.memoryUnit : 'MB';
+        paramModel.value.memoryUnit = res.data.memoryUnit !== '' ? res.data.memoryUnit : 'M';
         paramModel.value.allowPort = res.data.allowPort;
         paramModel.value.containerName = res.data.containerName;
         paramModel.value.advanced = false;
```

Comment:
The changes between lines 259 in both versions of the code indicate an inconsistency in handling memory units. The original code assigns "M" to `paramModel.value.memoryUnit` if `res.data.memoryUnit` is an empty string. However, the version with the comment has a typo, assigning 'MB' rather than 'M'. It would be advisable to correct this discrepancy to ensure consistent behavior with whatever logic expects regarding memory unit formatting. Here's how you can adjust it:

**Original Code snippet (line 259):**
```
-         paramModel.value.memoryUnit = res.data.memoryUnit !== '' ? res.data.memoryUnit : 'MB';
+         paramModel.value.memoryUnit = ... // Ensure consistent unit format
``` 

**Corrected Version:**
```
         paramModel.value.memoryUnit = res.data.memoryUnit !== '' ? res.data.memoryUnit : 'M'; 
         // Optionally, consider adding a function that formats memory units based on your requirements
         paramModel.value.memoryUnit = validateMemoryUnit(res.data.memoryUnit);
     ```
Remember to implement a corresponding validation function (`validateMemoryUnit`) if necessary to handle more complex scenarios where different input values might denote the same measurement unit.